server/sonar-server/src/main/java/org/sonar/server/rule/ws/ActiveRuleCompleter.java

```diff
@@ -97,7 +99,7 @@ void completeSearch(DbSession dbSession, RuleQuery query, List<RuleDto> rules, S
       }
     } else {
       // Load details of all active rules
-      List<ActiveRuleDto> activeRuleDtos = dbClient.activeRuleDao().selectByRuleIds(dbSession, Lists.transform(rules, RuleDto::getId));
+      List<ActiveRuleDto> activeRuleDtos = dbClient.activeRuleDao().selectByRuleIds(dbSession, query.getOrganizationUuid(), Lists.transform(rules, RuleDto::getId));
```

Comment:
I'm going to remove the `@CheckForNull` annotation of `query.getOrganizationUuid()`. It is not supposed to be null.